services/main.services.js

```diff
@@ -68,6 +68,7 @@ const postHandler = async (sender_psid, data) => {
     const product = await Product.findOne({ _id: productQuery });
     // send email to my account for buy request
     sendEmail(product);
+    handleMessage(sender_psid, data, "Your order has been placed, thank you for shopping with us!");
```

Comment:
Codacy has a fix for the issue: [Replace `sender_psid,·data,·"Your·order·has·been·placed,·thank·you·for·shopping·with·us!"` with `⏎······sender_psid,⏎······data,⏎······"Your·order·has·been·placed,·thank·you·for·shopping·with·us!"⏎····`](https://app.codacy.com/gh/ParmenidesSartre/Respond-Task1/pullRequest?prid=9934306)
```suggestion
    handleMessage(
```